account_cutoff_base/views/account_cutoff.xml

```diff
@@ -35,15 +33,15 @@
                 </div>
                 <group name="top">
                     <group name="general-params">
-                        <field name="cutoff_date" on_change="cutoff_date_onchange(type, cutoff_date, move_label, context)"/>
-                        <field name="total_cutoff_amount" widget="monetary" options="{'currency_field': 'company_currency_id'}"/>
+                        <field name="cutoff_date"/>
+                        <field name="total_cutoff_amount"/>
                         <field name="company_id" groups="base.group_multi_company" widget="selection" />
                         <field name="company_currency_id" invisible="1"/>
                     </group>
                     <group name="accounting-params">
-                        <field name="cutoff_journal_id"/>
-                        <field name="cutoff_account_id"/>
-                        <field name="move_label"/>
+                        <field name="cutoff_journal_id" required="1"/>
```

Comment:
OK, thanks for the explanation. A comment in the XML would save this question for future readers of the code.